com.interversehq.qView.yml

```diff
@@ -11,6 +11,112 @@ finish-args:
   - --share=network
   - --device=dri
 modules:
+  - name: kimageformats
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DBUILD_TESTING=OFF
+      - -DKIMAGEFORMATS_HEIF=ON
+    modules:
+      - name: libheif
+        config-opts:
+          - --disable-gdk-pixbuf
+        cleanup:
+          - /bin
+          - /share/man
+        modules:
+          - name: libde265
+            config-opts:
+              - --disable-sherlock265
+            cleanup:
+              - /bin
+            sources:
+              - type: archive
+                url: https://github.com/strukturag/libde265/releases/download/v1.0.8/libde265-1.0.8.tar.gz
+                sha256: 24c791dd334fa521762320ff54f0febfd3c09fc978880a8c5fbc40a88f21d905
+                x-checker-data:
+                  type: anitya
+                  project-id: 11239
+                  stable-only: true
+                  url-template: https://github.com/strukturag/libde265/releases/download/v$version/libde265-$version.tar.gz
+        sources:
+          - url: https://github.com/strukturag/libheif/releases/download/v1.12.0/libheif-1.12.0.tar.gz
+            sha256: e1ac2abb354fdc8ccdca71363ebad7503ad731c84022cf460837f0839e171718
+            type: archive
+            x-checker-data:
+              type: anitya
+              project-id: 64439
+              stable-only: true
+              url-template: https://github.com/strukturag/libheif/releases/download/v$version/libheif-$version.tar.gz
+
+      - name: libjxl
+        buildsystem: cmake-ninja
+        builddir: true
+        config-opts:
+          - -DBUILD_TESTING=OFF
+        cleanup:
+          - /bin
+        modules:
+          - name: highway
+            buildsystem: cmake
+            config-opts:
+              - -DBUILD_TESTING=OFF
+            sources:
+              - type: archive
+                url: https://github.com/google/highway/archive/refs/tags/0.15.0.tar.gz
+                sha256: 4bbd4439eae08cf038f1c5cc5d9ebc6a1a50f2c610c13a1483adccacfa24c150
+            x-checker-data:
+              type: anitya
+              project-id: 205809
+              stable-only: true
+              url-template: https://github.com/google/highway/archive/refs/tags/$version.tar.gz
+        sources:
+          - type: git
+            url: https://github.com/libjxl/libjxl.git
+            tag: v0.6.1
+            x-checker-data:
+              type: anitya
+              project-id: 232764
+              stable-only: true
+              tag-template: v$version
+
+      - name: libavif
+        buildsystem: cmake
+        config-opts:
+          - -DAVIF_BUILD_GDK_PIXBUF=OFF
```

Comment:
I am not sure about this libavif configuration.
I think there should be some AV1 decoders enabled: dav1d or AOM.

Reply:
libaom should be included in the runtime already.

Reply:
When `libaom` (except very old 1.0.0 version) is there, libavif can be configured with `-DAVIF_CODEC_AOM=ON`